pkg/services/folder/model.go

```diff
@@ -18,9 +18,10 @@ var ErrCircularReference = errutil.BadRequest("folder.circular-reference", errut
 var ErrTargetRegistrySrvConflict = errutil.Internal("folder.target-registry-srv-conflict")
 
 const (
-	GeneralFolderUID     = "general"
-	RootFolderUID        = ""
-	MaxNestedFolderDepth = 4
+	GeneralFolderUID      = "general"
+	RootFolderUID         = ""
+	MaxNestedFolderDepth  = 4
+	SharedWithMeFolderUID = "sharedwithme"
```

Comment:
I need to double check this. I talked to Sofia and seems, since folder uids are unique within the org, that might be possible to have folder with this UID. But it might introduce some mess. So I'll double check it.

Reply:
Done.